reflexible/conv2netcdf4/fortflex/build_FortFlex_py3.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+f2py3 -m FortFlex -h FortFlex.pyf FortFlex.f
+f2py3 -c --fcompiler=gfortran FortFlex.pyf FortFlex.f
```

Comment:
Based on [upstream](https://github.com/numpy/numpy/blob/master/numpy/f2py/setup.py#L57), the only difference between `f2py` and `f2py3` besides their name is which python interpreter executes them (i.e. Python 2 vs. Python 3). Now different distributions will have different interpreter names (in fact, `python` which produces `f2py` could even be a Python 3 interpreter), and I don't expect this to work for all distributions, but I believe that `python3` (which produces `f2py3`) is somewhat standard for generic Python 3. An alternative would be to invoke `f2py` explicitly using the Python 3 interpreter, but this is inconsistent with the other FortFlex build script and requires some guessing or searching for the name of the Python 3 interpreter (no different than guessing the name of `f2py`).

The user is of course not required to use this script to build the Python 3 FortFlex module, but I think this is probably generic enough to work for most users.